Sweeper/View Controllers/PinsMapViewController.swift

```diff
@@ -54,18 +54,6 @@ class PinsMapViewController: UIViewController {
             }
         }
     }
-
-    @IBAction func onPost(_ sender: UIBarButtonItem) {
```

Comment:
Did you move this to somewhere else?

Reply:
We only access the "Post" from the tab bar controller now.

Reply:
oh, right! LGTM